www/xkahj06/Projekt2/js/main.js

```diff
@@ -0,0 +1,318 @@
+               // console.log(chapterCount);
+               // console.log('opraveno');
+            }; 
+
+            
+            if (isNaN(chapterCount)) {
+                chapterCount=chapterCount.trim();
+                while((chapterCount.length>1)&&isNaN(chapterCount)){
+                   //console.log(chapterCount);
+                    chapterCount=chapterCount.substring(0 , chapterCount.length-1);
+                };
+            }
+
+            if (isNaN(chapterCount)) {
+                console.log(chapterCount);
+                chapterCount = -1;
+                //console.log("pokazilo se to");
+                
+            } else {
+            };
+            chapterCount=Number(chapterCount);
+           // console.log('vol '+ volume +' ch '+chapterCount);
+
+            if (((theHighestCHCount < chapterCount) && (volume == maxvolume))||(volume>maxvolume)) {
+                maxvolume = volume;
+                theHighestCHCount = chapterCount;
+                theHighestCHDetails[0] = title;
+                theHighestCHDetails[1] = url;
+                theHighestCHDetails[2] = volume;
+                theHighestCHDetails[3] = theHighestCHCount;
+               // console.log('probehlo to');
+            };
+        };
+
+    });
+
+
+
+    //console.log('nejvysi chapter je ' + theHighestCHCount);
+
+
+
+    displayTitles(rawTitles, theHighestCHCount, theHighestCHDetails, novelName);
+
+};
+
+
+
+
+const displayTitles = (rawTitles, theHighestCHCount, theHighestCHDetails, novelName) => {
+    const container = document.getElementById('js-output');
+    var counter = document.getElementById('js-counter');
+
+    counter.innerHTML = Number(counter.innerHTML) + 1;
+    // console.log(counter.innerHTML);
+
+    //zjisteni zda to uz splnuje podminky pro upozorneni 10x nasobek novych kapitol zluta,2,5x nasobek fialova,1x nasobek modra
+    var zvyseni = ((theHighestCHCount - novelName[1])+ ((theHighestCHDetails[2]-1)*100)) / novelName[2];
+        if (zvyseni > 5) {
+            extraColour = " violet";
+        } else {
+            extraColour = " blue";
+        };
+
+    
+
+    if (zvyseni < 1) {
+        extraColour = " ";
+    };
+
+if(theHighestCHDetails[2]>0){
+    theHighestCHCount= 'vol '+theHighestCHDetails[2]+' ch '+theHighestCHCount;
+};
+
+
+
+    newHTML = `<!--START${counter.innerHTML}-->
+      <div class="informations">
+            <div class="profile__detail">
+                <div class="profile__detailName">${novelName[0]}</div>
+                <div class="profile__detailValue">${novelName[1]}</div>
+                <div class="profile__detailValue${extraColour}"><a href=${theHighestCHDetails[1]}>${theHighestCHCount}</a></div>
+                <div class="profile__detailValue">${novelName[2]}</div>
+                <div class="profile__detailValue"><button class = "button button--submitSearch" onclick="deleteteLine(${counter.innerHTML})">Delete Novel</button></div>
```

Comment:
tohle je inline JS, -> predelat

Reply:
jako, že v Ajaxu by taky nemělo být inline JS? to mám volat z css? nebo odkud?

Reply:
nebo udělat nějaký event listener? uufff

Reply:
zvládl jsem to, chtělo to pouze 4 hodiny zkoušení :D..uff jdu spát